site/pages/[locale]/sign-message.js

```diff
@@ -63,48 +66,45 @@ const SignMessageForm = function () {
   }
 
   return (
-    <>
-      <div className="flex flex-wrap justify-center mt-10 mx-auto w-full max-w-lg space-y-4">
-        <textarea
-          className="w-full border-2"
-          placeholder={t('message-placeholder')}
-          rows={5}
-          title={`${t('message')}:`}
-          {...messageInput}
-        />
-      </div>
-      <div className="mt-7.5 flex justify-center">
-        <Button {...signButton}>{t('sign').toUpperCase()}</Button>
-      </div>
+    <UtilFormBox title={t('sign-message')}>
+      <TextArea
+        className="mb-"
+        placeholder={t('message-placeholder')}
+        rows={5}
+        title={t('message')}
+        {...messageInput}
+      />
+      <Button {...signButton}>{t('sign')}</Button>
       {signature && (
-        <div className="flex-wrap mt-6 w-full text-center text-sm">
-          Signature:
-          <br />
-          {signature}
+        <div className="mt-6 break-all">
+          <InputTitle>Signature</InputTitle>
```

Comment:
Same here, missing translation (we can also fix this later on in Upstream, if you prefer to create an issue)

Reply:
I opened a separated PR with this: https://github.com/hemilabs/pure.finance/pull/3